### ISSUE TITLE
NPE when trying to create a wallet with wallet-tool

### ISSUE THREAD

**Issue by schildbach** [bitcoinj/bitcoinj#3239]
On current master, I get this:

```
/bitcoinj/wallettool (master)$ ./build/install/wallet-tool/bin/wallet-tool create --wallet=test.wallet
java.lang.NullPointerException
	at org.bitcoinj.wallet.Wallet.saveToFile(Wallet.java:1708)
	at org.bitcoinj.wallet.Wallet.saveToFile(Wallet.java:1761)
	at org.bitcoinj.wallettool.WalletTool.createWallet(WalletTool.java:1107)
	at org.bitcoinj.wallettool.WalletTool.call(WalletTool.java:375)
	at org.bitcoinj.wallettool.WalletTool.call(WalletTool.java:111)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.bitcoinj.wallettool.WalletTool.main(WalletTool.java:336)
```

When commenting out the check below, it works.

```
        if (!destParentFile.exists()) {
            throw new FileNotFoundException(destParentFile.getPath() + " (wallet directory not found)");
        }
```

And it leaves a tempfile lying around.

A path like this

```
./build/install/wallet-tool/bin/wallet-tool create --wallet=./test.wallet
```

works as expected.

**Comment by schildbach** [comment]
Quick check against 0.16: This is a regression.

**Comment by msgilligan** [comment]
Fixed in PR #3240
